gef-ui/server/utils/helpers.js

```diff
@@ -66,7 +66,7 @@ const ErrorMessagesMap = {
  * Maps validation errors so both Summary Error component and field component display the error messages correctly.
  * @param {import('../types/validation-error').ValidationError[] | import('../types/validation-error').ValidationError} errs - errors to be mapped
  * @param {string} href - the current page URL
- * @returns {import('../types/view-models/facility-end-date-view-model').ViewModelErrors | null}
+ * @returns {import('../types/view-models/view-model-errors').ViewModelErrors | null}
```

Comment:
could this be ViewModelErrors |  null (whatever the type is in the file being referenced)